containers/src/Kokkos_DynamicView.hpp

```diff
@@ -93,6 +241,13 @@ class DynamicView : public Kokkos::ViewTraits<DataType, P...> {
  public:
   using traits = Kokkos::ViewTraits<DataType, P...>;
 
+  using value_type   = typename traits::value_type;
+  using device_space = typename traits::memory_space;
+  using host_space =
+      typename Kokkos::Impl::HostMirror<device_space>::Space::memory_space;
+  using device_accessor = Impl::SmartMemoryAccessor<device_space, value_type>;
+  using host_accessor   = Impl::SmartMemoryAccessor<host_space, value_type>;
```

Comment:
might be worthwhile to think about a better name. Since this is just the outer array, it doesn't actually give you access to the primary data on the host after all or?

Reply:
Changed the name to `ChunkedArrayManager`. Let me know if you can think of something better!